connect_ext_ppr/schemas.py

```diff
@@ -68,7 +68,11 @@ class ConfigurationCreateSchema(NonNullSchema):
     file: FileSchema
 
 
-class ConfigurationSchema(NonNullSchema):
+class PPRCreateSchema(NonNullSchema):
```

Comment:
Description could be provided by user so it is required here. But I have it in my PR, so I will add it after merge.